### DISCUSSION TITLE
Debugging the recipe scraper

### DISCUSSION THREAD

**Discussion by slatkin** [mealie-recipes/mealie#4496] [General]
Hi! Sorry if this has been asked, but I could not find anything related to debugging the recipe scraper.

I'm migrating from another cloud site (cooked.wiki) and some URLs that were scraped fine there don't scrape well in Mealie. Before just reporting it I wanted to do some debugging on my own and was wondering what tools there were to accomplish this? Are there logs or a way I can trigger scraping manually?

**Comment by michael-genson** [comment]
There's an internal tool at `https://your-mealie-url.com/g/your-group-slug/r/create/debug`
You can get there by opening any "create" page and choosing it from the dropdown.
